config/crd/bases/awx.ansible.com_awxs.yaml

```diff
@@ -497,7 +497,11 @@ spec:
                   type: boolean
                   default: true
                 set_self_labels:
-                  description: Maintain some of the recommended `app.kubernetes.io/*` labels on the resource (self)
+                  description: Maintain some of the recommended `app.kubernetes.io/*` labels on the custom resource object
+                  type: boolean
+                  default: true
+                set_self_status:
```

Comment:
@shanemcd we might want to add a bit more of a warning here in the description.  If users set this to false and are using the operator via `make deploy` or OLM/OKD, the reconciliation loop will just keep on running. 

Also, can you add an entry in the CSV template?  By default, a new field on the CRD will be added to the form (and not in even in the Advanced section).  

Something like this should work (config/manifests/bases/awx-operator.clusterserviceversion.yaml)
```
      - displayName: Set AWX statuses
        path: set_self_status
        x-descriptors:
        - urn:alm:descriptor:com.tectonic.ui:hidden
```

Similarly, for the AWXBackup and AWXRestore CR's.

Otherwise, I think this is a great addition.  Once the conflicts from the sdk upgrade are resolved, it has my :heavy_check_mark: